helper-scripts/options/devOps.json

```diff
@@ -1,20 +1,22 @@
 {
     "modelName": "devOps",
-    "numConsecutiveDays": 5,
-    "supportStartHour": 3,
-    "supportEndHour": 27,
-    "slotsInDay": 27,
+    "numDays": 5,
+    "startHour": 3,
+    "endHour": 27,
+    "hoursInDay": 27,
```

Comment:
Is there a reason why `hoursInDay` is set to `27` for `balenaio` & `devOps`? I see that it is set to 24 for `productOS`. Wouldn't this affect how we compute the number of slots given `config["slots_in_day"] = int(input_json["options"]["hoursInDay"] * 2)`?

Reply:
@cmfcruz The value of 27 indicates that the "support day" ends 3 hours after midnight, which means that availabilities from the start of the next day are needed to "extend" the given day.

Reply:
See the line
```
day.push(agent.availableSlots[followingDay][i]);
```
in `gsheets.ts`.